src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -339,4 +358,59 @@ private function announce($subject, Transition $initialTransition, Marking $mark
             $this->dispatcher->dispatch(sprintf('workflow.%s.announce.%s', $this->name, $transition->getName()), $event);
         }
     }
+
+    /**
+     * @param string $transitionName
+     *
+     * @return Transition[]
+     */
+    private function getTransitionsByName(string $transitionName)
```

Comment:
This is `private`, because I find the fact, that this method returns a collection instead of a `Transition|null` too confusing to make this method `public`,